Atomic/syscontainers.py

```diff
@@ -1822,7 +1822,7 @@ def _check_system_oci_image(self, repo, img, upgrade):
         if not upgrade and current_rev[1]:
             return False
 
-        can_use_skopeo_copy = str(util.check_output([util.SKOPEO_PATH, "copy", "--help"])).find("ostree") >= 0
+        can_use_skopeo_copy = str(util.check_output([util.SKOPEO_PATH, "copy", "--help"])).decode().find("ostree") >= 0
```

Comment:
Almost ... the ``str()`` is still going to cause a problem. Remove it and you should be good:

```
$ python3 -c "str(b'test').decode()"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'str' object has no attribute 'decode'
```

```
$ python3 -c "print(b'test'.decode())"
test
$ python2 -c "print('test'.decode())"
test
```

Reply:
oops, dropped it and pushed a new verison